tardis/model/base.py

```diff
@@ -490,23 +519,39 @@ def from_config(cls, config):
             )
         else:
             t_radiative = None
+        logger.debug(f"Radiative Temperature : {t_radiative}")
```

Comment:
`UnboundLocalError: local variable 't_radiative' referenced before assignment` I believe this is happening here (test failure)

Reply:
Sure, this PR has many errors. I will fix them as I go through the code once again making it ready for review 😄

Reply:
Yes, you are absolutely correct @andrewfullard. There is a complex code there which assigns `t_radiative` in multiple locations based on the conditions that are passed to the CSVY. I have fixed it hopefully & will push the changes :)